<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Everything</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_find">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>5</height>
        </size>
       </property>
       <property name="text">
        <string>üîç</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QListWidget" name="listWidget_result"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_select">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>5</height>
        </size>
       </property>
       <property name="text">
        <string>ÊµèËßà</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_dir">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Êü•ÊâæË∑ØÂæÑ</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_filePattern">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Êñá‰ª∂Pattern</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
